app/models/campaign.rb

```diff
@@ -412,17 +412,10 @@ def sanitize_assigned_property_ids
   end
 
   def validate_url
-    uri = begin
-            URI.parse(url)
-          rescue
-            nil
-          end
-    case uri
-    when URI::HTTP, URI::HTTPS
-      response = Typhoeus.get(uri.to_s, followlocation: true)
-      errors[:url] << "is invalid" unless response.success?
-    else errors[:url] << "is invalid"
-    end
+    self.url = url.to_s.strip
+    URI.parse(url)
+  rescue
+    errors[:url] << "is invalid"
```

Comment:
Any reason you are shoveling this in? I am not seeing why this would be an array..